build.sbt

```diff
@@ -54,7 +70,7 @@ lazy val commonSettings =  Seq(
     organization := "com.mfglabs"
   , version := "0.4.6-rc3"
   , isSnapshot := false
-  , crossScalaVersions := Seq("2.11.12", "2.12.6")
+  , crossScalaVersions := Seq("2.11.12", "2.12.8")
```

Comment:
FYI 2.12.9 came out a couple of days ago (cf https://www.scala-lang.org/news/2.12.9)

Reply:
Thanks. Unfortunately, the splain plugin is not yet available for 2.12.9.